### PR TITLE
v0.9.0

### PR DESCRIPTION
**Pull request by nanu-c**

### CODE REVIEW

File: flatpak/org.nanuc.Axolotl.appdata.xml [hunk 1]
```diff
@@ -21,6 +21,9 @@
         </p>
     </description>
     <releases>
+      <release version="0.8.9" date="2020-11-30">
```
Comment (Flaburgan) [inline flatpak/org.nanuc.Axolotl.appdata.xml:24]: It should be 0.9.0 there.

**Review by Flaburgan — CHANGES_REQUESTED** [review]

File: flatpak/org.nanuc.Axolotl.appdata.xml (hunk 1 above)
Comment (nanu-c) [reply flatpak/org.nanuc.Axolotl.appdata.xml]: i missed the appimage, too. It's the first release also for flatpack and appimage so it's new where to change the number :)

**Comment by Flaburgan** [comment]
There still is a reference of 0.8.9 in `docs/INSTALL.md` line 69, do we want to change this too?

**Comment by nanu-c** [comment]
We want it change there, too, but we need a script or a checklist for releases ...

**Review by Flaburgan — APPROVED** [review]

**Pull request merged**